docs/hub/spaces-embed.md

```diff
@@ -0,0 +1,52 @@
+# Embed your Space in another website
+
+Once your Space is up and running you might wish to embed it in a website or in your blog. 
+Embedding or sharing your Space is a great way to allow your audience to interact with your work and demonstrations without requiring any setup on their side.
```

Comment:
Maybe mention this only works with public Spaces

Reply:
Added:
> To embed a Space its visibility needs to be public.